### PR TITLE
Adds no standard view support; Fixes #205

### LINKED ISSUE #205: Add support for NoStandardView field option for time fields

### PR DESCRIPTION
**Pull request by yuce**
Fixes #205

### CODE REVIEW

File: orm.go [hunk 1]
```diff
@@ -648,10 +654,13 @@ func OptFieldTypeInt(min int64, max int64) FieldOption {
 }
 
 // OptFieldTypeTime adds a time field.
-func OptFieldTypeTime(quantum TimeQuantum) FieldOption {
+func OptFieldTypeTime(quantum TimeQuantum, opts ...bool) FieldOption {
 	return func(options *FieldOptions) {
 		options.fieldType = FieldTypeTime
 		options.timeQuantum = quantum
+		if len(opts) > 0 && opts[0] == true {
```
[the remark this answers is not in the document]
Comment (yuce) [reply orm.go]: Yeah, that could be written that way too :)

File: client.go [hunk 2]
```diff
@@ -817,15 +817,24 @@ func columnsToBitmap(shardWidth uint64, records []Record) viewImports {
 	return map[string]*roaring.Bitmap{"": bmp}
 }
 
-func columnsToBitmapTimeField(quantum TimeQuantum, shardWidth uint64, records []Record) viewImports {
-	views := viewImports{
-		"": roaring.NewBitmap(),
+func columnsToBitmapTimeField(quantum TimeQuantum, shardWidth uint64, records []Record, noStandardView bool) viewImports {
+	var standard *roaring.Bitmap
```
[the remark this answers is not in the document]
Comment (yuce) [reply client.go]: Yeah, I've introduced `standard` to not access it using the array subscript.

**Comment by jaffee** [comment]
Is this waiting on anything to be approved? FWIW I used it in some of my testing and it seems to work.

**Comment by travisturner** [comment]
Well, I provided some feedback because I thought the code was unnecessarily complex (or at least wordy) in places. If anyone else wants to review and veto those suggestions I'm fine with that.

**Comment by yuce** [comment]
@travisturner I've updated the code according to your `views` comment. I didn't remove `standard`, since I think it makes the code easier to read.

**Pull request merged**